federatedscope/core/trainers/torch_trainer.py

```diff
@@ -68,8 +69,10 @@ def update(self, model_parameters, strict=False):
         """
         for key in model_parameters:
             model_parameters[key] = param2tensor(model_parameters[key])
-        self.ctx.model.load_state_dict(self._param_filter(model_parameters),
-                                       strict=strict)
+        # Due to lazy load, we merge two state dict
```

Comment:
When would lazy load be used? And can `merge_param_dict` works well when not using lazy load?

Reply:
This won't affect when not using lazy load.

Reply:
Maybe explain the reason in more detail. `From torch_geometric version 2.03 ...`